edx_rest_framework_extensions/__init__.py

```diff
@@ -1,3 +1,3 @@
 """ edx Django REST Framework extensions. """
 
-__version__ = '2.0.3'  # pragma: no cover
+__version__ = '2.0.4'  # pragma: no cover
```

Comment:
I know it is a tiny change, but I usually reserve the last for bug fixes. That would make this 2.1.0.

Reply:
can do